tests/driver_blemin/README.md

```diff
@@ -0,0 +1,115 @@
+Expected result
+===============
+The test will initialize all basic networking functionality including the
+minimal NRF51822 BLE radio driver and run the shell providing netif shell
+commands (**blesnd** and **ifconfig**). The transmitted BLE packets can
+be captured and verified using SDR hardware like Ubertooth One or
+HackRF.
+
+Background
+==========
+Use the shell commands to test the link layer functionality of the minimal
+NRF51822 BLE radio driver (nrfmin). Further information about the BLE protocol
+standard and excpected packet data formats can be found within the official
+**BLUETOOTH SPECIFICATION Version 4.2 [Vol 6, Part B]**
+
+Usage
+=====
+The **blesnd** shell command is used to send BLE packets via the NRF51822
+BLE radio driver. The following list of BLE Advertising Channel packet types
+are supported:
+- ADV_IND
+- ADV_DIRECT_IND
+- ADV_NONCONN_IND
+- SCAN_REQ
+- SCAN_RSP
+- ADV_SCAN_IND_TYPE
+
+The **ifconfig** shell command is used to set the requested BLE Advertising
+Address, BLE Channel number and BLE Access Address (optional).
+
+Examples
+========
+(Transmitted BLE packets were captured using Ubertooth One)
+
+Shell commnad: `ifconfig 5 set adv_addr c2:5e:72:69:6f:74`
```

Comment:
Running the shell application the ble interface number is 4. I thinki it is the same on your machine, right?